### ISSUE TITLE
EBV viremia as a non-qualified alternative to "Persistent EBV viremia"

### ISSUE THREAD

**Issue by BenSolomon** [obophenotype/human-phenotype-ontology#11018]
**Preferred term label:**
Positive EBV PCR test in the blood circulation

**Synonyms**
EBV viremia

**Definition (free text, please give PubMed ID)**
Evidence of active EBV infection based on nucleic acid detection

**Parent term (use hpo.jax.org/app)**
HP:0430067 - Positive bloodstream nucleic acid pathogen test

Not a parent term, but also related to HP:0020072 - Persistent EBV viremia. An important distinction because many inborn errors of immunity are associated with persistent EBV viremia, but individual cases that otherwise fit the broader disease phenotype may have a history of EBV infection that is not yet persistent. Including a term that allows for documentation of EBV viremia without needing to reach the threshold of persistence would allow better phenotypic documentation of these cases. 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
All the same disease already associated with HP:0020072 

**Your nano-attribution (ORCID)**
0000-0003-1009-7312
